src/Datadog.Trace/ExtensionMethods/SpanExtensions.cs

```diff
@@ -48,7 +49,8 @@ public static void AddTagsFromDbCommand(this Span span, IDbCommand command)
             string resourceName,
             string method,
             string host,
-            string httpUrl)
+            string httpUrl,
+            IEnumerable<KeyValuePair<string, string>> headerTags)
```

Comment:
Consider renaming parameter `headerTags` to `tags`, since at this point they're not tied to headers in any way. Also the KVPs in this `headerTags` have a very different meaning (tag name, tag value) than the `headerTags` in `SpanContextPropagator.ExtractHeaderTags()` (header name, tag name).

Reply:
Good idea, done in d73d7d0729eab2a56a03803a4b4f2a55fc73b673